spec/features/customer_details_spec.rb

```diff
@@ -0,0 +1,100 @@
+Capybara.app = Sinatra::Application
+
+describe 'customer details', type: :feature do
+  it 'will accept a customer name' do
+    visit '/customer-details'
+    within('form') do
+      fill_in 'customer_name', with: 'Larry'
+    end
+    click_button 'Submit'
+    expect(page).to have_no_content 'Missing Customer Name'
+  end
+
+  it 'will not accept an invalid phone number' do
+    visit '/customer-details'
+    within('form') do
+      fill_in 'shipping_phone_number', with: '0123453'
+    end
+    click_button 'Submit'
+    expect(page).to have_content 'Invalid Shipping Phone Number'
+  end
+
+  it 'will not accept an invalid postcode number' do
```

Comment:
maybe to have consistent 'it' messages, we could call all the 'it' messages start from 'can ..... '